airflow/bin/cli.py

```diff
@@ -600,6 +600,10 @@ def get_parser():
         type=int,
         help="Number of threads to run the webserver on")
     parser_webserver.add_argument(
+        "-k", "--workerclass",
+        default=configuration.get('webserver', 'WORKER_CLASS'),
```

Comment:
`choices=['sync', 'eventlet', 'gevent', 'tornado'],` We can constrain this since those are the only options.